charts/keycloakx/values.yaml

```diff
@@ -298,7 +298,7 @@ ingress:
         host: '{{ .Release.Name }}.keycloak.example.com'
         # Paths for the host
         paths:
-          - path: /auth/admin/
+          - path: '{{ trimSuffix "/" .Values.http.relativePath}}/admin'
```

Comment:
You might also need to adjust the following line https://github.com/codecentric/helm-charts/blob/d14250633937796653405b12c5699bf6362b7d18/charts/keycloakx/templates/ingress.yaml#L100 to look like `{{ tpl .path $ | quote }}`. Otherwise this template syntax will just be rendered as a string.

Reply:
Fixed